src/nih_wayfinding/app/scripts/views/locations/add-location/location-profile/location-profile-controller.js

```diff
@@ -82,13 +82,26 @@
          */
         function onGeocoderResponse(data) {
             if (data.length) { // If non-empty result
-                // Add geometric features to location tracker
-                ctl.user.setTempLocationProperty('feature', data[0]);
+                // check if location is within graph bounds
+                MapControl.getGraphBounds().then(function(geojson) {
```

Comment:
It probably makes more sense to perform this check in the geocoder find response, and reject the promise with an error message there. That way, we get this bounds check both here and in the typeahead on the locations view.